vm_supervisor/__init__.py

```diff
@@ -1 +1,12 @@
+from subprocess import check_output
```

Comment:
Note that you have a build error, the CI cannot find this file when building the Debian package.